frontend/src/pages/CourseSelector/CourseSelector.tsx

```diff
@@ -70,7 +74,7 @@ const CourseSelector = () => {
     };
     resizerDiv?.addEventListener('mousedown', startResize);
     return () => resizerDiv?.removeEventListener('mousedown', startResize);
-  }, []);
+  }, [showedNotif, coursesQuery.isSuccess, coursesQuery.data]);
```

Comment:
could we put the notification in its own useEffect? keeping them compartmentalized when they are doing different things and have different dependencies is good

Reply:
Fixed